tests/scanner/audit/external_project_access_engine_test.py

```diff
@@ -211,27 +227,95 @@ class ExternalProjectAccessRuleTest(ForsetiTestCase):
 
     def test_single_item_in_rule_match(self):
         """Test no violations are found with single item in rule"""
+        processed_rule = {
+            'ancestor_resources': [Organization('7890')],
+        }
+
         rule = engine_module.Rule(rule_name='test_single_item_in_rule_match',
                                   rule_index=0,
-                                  rules=[Organization('7890')])
+                                  rules=processed_rule)
+        violation = rule.find_violation('user1@example.com',
+                                        self.TEST_ANCESTORS)
+        self.assertIsNone(violation)
+
+    def test_single_item_in_rule_match_with_user(self):
+        """Test no violations are found with single item in rule"""
```

Comment:
This description should be a bit different for this test, i.e. add "with user".  Can you please update?